src/AppBundle/Form/PostType.php

```diff
@@ -39,7 +39,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         //     $builder->add('title', null, array('required' => false, ...));
 
         $builder
-            ->add('title', null, array('label' => 'label.title'))
+            ->add('title', null, array(
+                'attr' => array('autofocus' => true),
```

Comment:
I prefer to keep attributes on the form view, but I would like to knows others intentions